lib/testScenario/postmanCollectionRunnerClient.ts

```diff
@@ -508,6 +534,16 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
           console.log("collection run complete!");
         }
       )
+      .on("beforeItem", async function (this: any, _err, _summary) {
+        if (!_err) {
+          runtimeEnvManager.save(_summary.item.name, this, "beforeStep");
+        }
+      })
+      .on("item", async function (this: any, _err, _summary) {
+        if (!_err) {
+          runtimeEnvManager.clean();
```

Comment:
Why do clean here

Reply:
clean should trigger when any of step is completed